Building libsoxr (as a dependency for Shairport-Sync) fails on 10.4.11 Tiger

Hello,

I am currently trying to adapt the brew-formula for Shairport-Sync for tigerbrew to get it running on my G3 iMac.

On of its dependencies is libsoxr - which already had a tigerbrew formula. 
When I try to install libsoxr it fails with the following output:

`Last 15 lines from /Users/jan/Library/Logs/Homebrew/libsoxr/02.make:
cd /tmp/libsoxr20200312-260-uiejkb/soxr-0.1.1-Source/src && /usr/local/bin/gcc-4.2  -DSOXR_LIB -DSOXR_VISIBILITY -Dsoxr_EXPORTS -I/tmp/libsoxr20200312-260-uiejkb/soxr-0.1.1-Source -I/tmp/libsoxr20200312-260-uiejkb/soxr-0.1.1-Source/src  -Os -w -pipe -mcpu=750 -faltivec -mmacosx-version-min=10.4  -isysroot /Developer/SDKs/MacOSX10.4u.sdk -mmacosx-version-min=10.4 -fPIC   -fvisibility=hidden -Wconversion -Wall -W -pedantic -Wundef -Wcast-align -Wpointer-arith -Wno-long-long -Wnested-externs -Wmissing-prototypes -Wstrict-prototypes -o CMakeFiles/soxr.dir/rate32.c.o   -c /tmp/libsoxr20200312-260-uiejkb/soxr-0.1.1-Source/src/rate32.c
[ 30%] Building C object src/CMakeFiles/soxr.dir/fft4g32.c.o
cd /tmp/libsoxr20200312-260-uiejkb/soxr-0.1.1-Source/src && /usr/local/bin/gcc-4.2  -DSOXR_LIB -DSOXR_VISIBILITY -Dsoxr_EXPORTS -I/tmp/libsoxr20200312-260-uiejkb/soxr-0.1.1-Source -I/tmp/libsoxr20200312-260-uiejkb/soxr-0.1.1-Source/src  -Os -w -pipe -mcpu=750 -faltivec -mmacosx-version-min=10.4  -isysroot /Developer/SDKs/MacOSX10.4u.sdk -mmacosx-version-min=10.4 -fPIC   -fvisibility=hidden -Wconversion -Wall -W -pedantic -Wundef -Wcast-align -Wpointer-arith -Wno-long-long -Wnested-externs -Wmissing-prototypes -Wstrict-prototypes -o CMakeFiles/soxr.dir/fft4g32.c.o   -c /tmp/libsoxr20200312-260-uiejkb/soxr-0.1.1-Source/src/fft4g32.c
[ 34%] Building C object src/CMakeFiles/soxr.dir/vr32.c.o
cd /tmp/libsoxr20200312-260-uiejkb/soxr-0.1.1-Source/src && /usr/local/bin/gcc-4.2  -DSOXR_LIB -DSOXR_VISIBILITY -Dsoxr_EXPORTS -I/tmp/libsoxr20200312-260-uiejkb/soxr-0.1.1-Source -I/tmp/libsoxr20200312-260-uiejkb/soxr-0.1.1-Source/src  -Os -w -pipe -mcpu=750 -faltivec -mmacosx-version-min=10.4  -isysroot /Developer/SDKs/MacOSX10.4u.sdk -mmacosx-version-min=10.4 -fPIC   -fvisibility=hidden -Wconversion -Wall -W -pedantic -Wundef -Wcast-align -Wpointer-arith -Wno-long-long -Wnested-externs -Wmissing-prototypes -Wstrict-prototypes -o CMakeFiles/soxr.dir/vr32.c.o   -c /tmp/libsoxr20200312-260-uiejkb/soxr-0.1.1-Source/src/vr32.c
[ 39%] Building C object src/CMakeFiles/soxr.dir/rate64.c.o
cd /tmp/libsoxr20200312-260-uiejkb/soxr-0.1.1-Source/src && /usr/local/bin/gcc-4.2  -DSOXR_LIB -DSOXR_VISIBILITY -Dsoxr_EXPORTS -I/tmp/libsoxr20200312-260-uiejkb/soxr-0.1.1-Source -I/tmp/libsoxr20200312-260-uiejkb/soxr-0.1.1-Source/src  -Os -w -pipe -mcpu=750 -faltivec -mmacosx-version-min=10.4  -isysroot /Developer/SDKs/MacOSX10.4u.sdk -mmacosx-version-min=10.4 -fPIC   -fvisibility=hidden -Wconversion -Wall -W -pedantic -Wundef -Wcast-align -Wpointer-arith -Wno-long-long -Wnested-externs -Wmissing-prototypes -Wstrict-prototypes -o CMakeFiles/soxr.dir/rate64.c.o   -c /tmp/libsoxr20200312-260-uiejkb/soxr-0.1.1-Source/src/rate64.c
[ 43%] Linking C shared library libsoxr.dylib
cd /tmp/libsoxr20200312-260-uiejkb/soxr-0.1.1-Source/src && /usr/local/Cellar/cmake/3.6.3/bin/cmake -E cmake_link_script CMakeFiles/soxr.dir/link.txt --verbose=1
/usr/local/bin/gcc-4.2  -Os  -pipe -mcpu=750 -faltivec -mmacosx-version-min=10.4  -isysroot /Developer/SDKs/MacOSX10.4u.sdk -mmacosx-version-min=10.4 -dynamiclib -Wl,-headerpad_max_install_names  -L/usr/local/lib -F/usr/local/Frameworks -Wl,-headerpad_max_install_names -s -current_version 0.1.0 -o libsoxr.0.1.0.dylib -install_name /tmp/libsoxr20200312-260-uiejkb/soxr-0.1.1-Source/src/libsoxr.0.dylib CMakeFiles/soxr.dir/soxr.c.o CMakeFiles/soxr.dir/data-io.c.o CMakeFiles/soxr.dir/dbesi0.c.o CMakeFiles/soxr.dir/filter.c.o CMakeFiles/soxr.dir/fft4g64.c.o CMakeFiles/soxr.dir/rate32.c.o CMakeFiles/soxr.dir/fft4g32.c.o CMakeFiles/soxr.dir/vr32.c.o CMakeFiles/soxr.dir/rate64.c.o -lm 
/usr/bin/ld: can't use -s with -dylib (file must contain at least global symbols, for maximum stripping use -x)
collect2: ld returned 1 exit status
make[2]: *** [src/libsoxr.0.1.0.dylib] Error 1
make[1]: *** [src/CMakeFiles/soxr.dir/all] Error 2
make: *** [all] Error 2
`

Has somebody maybe an idea how to fix that?

--- a/Library/Formula/libsoxr.rb
+++ b/Library/Formula/libsoxr.rb
@@ -13,6 +13,7 @@ class Libsoxr < Formula
     sha1 "1da33c9683edeaa55a5d79dc08efd40e79a13b53" => :mountain_lion
   end
 
+  depends_on :ld64
   depends_on "cmake" => :build
 
   conflicts_with "sox", :because => "Sox contains soxr. Soxr is purely the resampler."

